source/includes/steps-general-connection-tab.yaml

```diff
@@ -0,0 +1,66 @@
+---
+title: Click on :guilabel:`Advanced Connection Options`.
+level: 4
+ref: compass-advanced-connection
+content: |
+  .. figure:: /images/compass/advanced-connection-options.png
+     :figwidth: 690px
+     :alt: New Advanced Connection Options
+---
+title: Select :guilabel:`General`.
+level: 4
+ref: select-general-tab
+content: |
+  .. procedure::
+  
+     .. step:: Select a Connection String Scheme.
+
+        You can select from the following two options:
+
+        .. list-table::
+           :header-rows: 1
+           :class: border-table
+
+           * - Connection String Scheme
+             - Definition
+
+           * - mongodb
+             - :manual:`Standard Connection String Format </reference/connection-string/>`. 
+               The standard format of the MongoDB connection URI is used to 
+               connect to a MongoDB deployment: standalone, replica set, or a sharded 
+               cluster. 
+
+           * - mongodb+srv 
+             - DNS Seed List Connection Format. The +srv indicates to the client that 
+               the hostname that follows corresponds to a 
+               :manual:`DNS SRV record </reference/connection-string/>`.
+
+     .. step:: Enter a Hostname.
+
+        Hostname of the machine where the deployment is running.
+
+        If you are running your deployment locally, this value is localhost. 
+        If you are connecting to an Atlas cluster, you can get your hostname 
+        from your cluster detail view in Atlas.
+
+        If you are not sure of your hostname, contact your Database 
+        Administrator for information.
+
+     .. step:: Enable Direct Connection. 
+
+        When you use the ``mongodb`` Connection String Scheme, you have the 
+        option to enable a :guilabel:`Direct Connection`. This specifies 
+        whether to force dispatch all operations to the specified host.
```

Comment:
"force dispatch all operations" is a little confusing / jargon-y I think. Maybe we can simplify this a bit.

"When you enable this setting, Compass runs all operations on the specified host."

It could also be worth noting that by default, when you specify directConnection and connect to a secondary member of the replica set, your write operations fail because it is not the primary member.

More info on direct connection here for context: https://docs.mongodb.com/drivers/node/current/fundamentals/connection/#direct-connection